src/main/java/com/websafe/ccmsbe/controller/PrivacyRegulationController.java

```diff
@@ -0,0 +1,58 @@
+package com.websafe.ccmsbe.controller;
+
+
+import com.websafe.ccmsbe.entity.PrivacyRegulation;
+import com.websafe.ccmsbe.service.PrivacyRegulationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin("http://localhost:3000")
```

Comment:
We need to get this value depending on the environment. For example, for local development -> localhost:3000, for production -> backend hosted URL etc. We can do this using spring profiles.

Ref: https://www.baeldung.com/spring-profiles